cosmos-gui/src/mysql_driver.js

```diff
@@ -37,89 +37,99 @@ var connection = mysql.createConnection({
     database: mysqlConfig.database
 });
 
-function connect(callback) {
-    connection.connect(function (error) {
+function addUser(idm_username, username, password, hdfsQuota, callback) {
+    pool.getConnection(function(error, connection) {
         if (error) {
             callback(error);
         } else {
-            logger.info('Connected to http://' + mysqlConfig.host + ':' + mysqlConfig.port + '/' +
-                mysqlConfig.database);
-            callback(null);
+            var query = connection.query(
+                'INSERT INTO cosmos_user (idm_username, username, password, hdfs_quota) ' +
+                'VALUES (?, ?, ?, ?)',
+                [idm_username, username, password, hdfsQuota],
+                function (error, result) {
+                    if (error) {
+                        callback(error)
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cosmos/commit/7416160a9a4606c741cd174e8a8f954464c67747